offerzen/global/employers/employers-signup-member-alerts@1.0.js

```diff
@@ -0,0 +1,45 @@
+!(function () {
+  window.$loaded(function (window, document, $, undefined) {
+    const params = new URLSearchParams(window.location.search);
+    $('.js-member-alert-message').val('');
+
+    if (params.has('member')) {
+      const alert = params.get('member');
+      //show the relevant message based on alert
+      switch (alert) {
+        case 'invalid_token':
+          $('.js-member-alert-message').text(
+            'This invite has been used already.'
+          );
+          $('.js-member-alert').show();
+          break;
+        case 'already_exists':
+          $('.js-member-alert-message').text(
+            "You've already been added as a member."
+          );
+          $('.js-member-alert').show();
+          break;
+        default:
+          return;
+      }
+
+      //clean url
+      var cleanSearch = window.location.search
+        .replace(/member[^&]+&?/g, '')
+        .replace(/&$/, '')
+        .replace(/^\?$/, '');
+
+      window.history.replaceState(
+        {},
+        '',
+        window.location.pathname + cleanSearch
+      );
```

Comment:
It's better to construct the URL Params using the built-in JS calls:

```suggestion
const searchParams = new URLSearchParams(window.location.search)
searchParams.delete('member')
const url = new URL(window.location) 
url.searchParams = searchParams
window.history.replaceState({}, '', url.pathname + url.search); // includes `?`
```